keps/NNNN-kep-template/README.md

```diff
@@ -44,6 +44,10 @@ When editing KEPS, aim for tightly-scoped, single-topic PRs to keep discussions
 focused. If you disagree with what is already in a document, open a new PR
 with suggested changes.
 
+Wrap long lines instead of using one line per paragraph. When updating, change
```

Comment:
How long is "long"? We should probably agree to one recommendation versus adhoc per-KEP reviewer.

Reply:
80